compiler/src/iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -104,6 +104,40 @@ class TensorDimTrackingRewriter : public IRRewriter {
  private:
   SmallPtrSet<Operation *, 16> dimOps;
 };
+
+/// A rewriter that keeps track of all operations when tracker is on
+class AllOpsTrackingRewriter : public TensorDimTrackingRewriter {
```

Comment:
I'm not sure I understand your comment.

 I created this class to track all the Ops we generate for the workload.  These Ops are generated in dispatch.region since the rootOp is there. So I'm tracking them and moving them outside of the dispatch.region

Reply:
I am asking why this extra class is needed. Looking at its uses, the only method called seems to be `getGeneratedOperations` and there only `tensor.dim` ops are handled... So trying to understand the need for all the operations being tracked.